migrations/V25__Add_System_Intake_Lifecycle_Fields.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE system_intake ADD COLUMN lcid text;
+ALTER TABLE system_intake
+  ADD CONSTRAINT lcid_check CHECK (lcid ~ '^[0-9]{6}$');
+CREATE INDEX lcid_idx ON system_intake (lcid) WHERE LENGTH(lcid)>0;
+
+ALTER TABLE system_intake ADD COLUMN expires_at timestamp with time zone;
```

Comment:
I would be tempted to prefix this column (and next steps) with `lcid`. It might make it more clear what this data is for.